src/backend/renderer.js

```diff
@@ -707,6 +707,16 @@ export function attach(
   function enqueueUnmount(fiber) {
     const isRoot = fiber.tag === HostRoot;
     const primaryFiber = getPrimaryFiber(fiber);
+    if (!fiberToIDMap.has(primaryFiber)) {
```

Comment:
We need this because React informs us about _every_ unmount, including "not-conceptually-primary" Suspense children tree. However, our maps only "remember" things in the "conceptually-primary" tree.